plugins/generic/pln/PLNPlugin.inc.php

```diff
@@ -329,6 +329,12 @@ function manage($verb, $args, &$message, &$messageParams) {
 				$this->import('classes.form.PLNSettingsForm');
 				$form = new PLNSettingsForm($this, $journal->getId());
 				
+				$application =& PKPApplication::getApplication();
+				$products =& $application->getEnabledProducts('plugins.generic');
+				if( ! isset($products['acron']) && ! Config::getVar('scheduled_tasks', 'Off')) {
```

Comment:
The default value should be `false`, not `'Off'`, if you're expecting a `boolean` out the back.